### ISSUE TITLE
can not scroll in  chrome 74 or highter

### ISSUE THREAD

**Issue by gdutwyg** [BoxFactura/pulltorefresh.js#80]
# Bug report

**Current behavior:**
 in the chorme 74,the page I can slide down, but i can't slide up. And  it is normal if i use chrome 66 or lower
**Expected behavior:**
the page can slide down or slide up
**JSFiddle URL for demo with bug:**
https://jsbin.com/wisiligeke/1/edit?html,js,console,output
**Browsers affected:**
mobile chorme 74

**Comment by faelsoto** [comment]
@pateketrueke it's the issue we were talking about last week.

@gdutwyg we're looking into it, a quick fix would be to add a new parameter:

```
shouldPullToRefresh(){
   return !window.location.top;
}
```

**Comment by gdutwyg** [comment]
@faelsoto oh，it seems that the code can not fix it， and i add the code，also it is not normal in  chorme 66.

**Comment by faelsoto** [comment]
I'm sorry, I made up that property, don't know what I was thinking.

The fix as we have it on production is this:

```javascript
shouldPullToRefresh(){
   !window.scrollY;
}
```

**Comment by gdutwyg** [comment]
ok, it works. i hope you can fix it  in next version, thanks.

**Comment by allstar613** [comment]
Sorry, I added those code into my project.
It can scroll now, but can't not trigger onRefresh().
Any thing I miss to do ?

**Comment by faelsoto** [comment]
It's fixed, please download v0.1.19

**Comment by crusy** [comment]
> It's fixed, please download v0.1.19

It's not on npm yet, or is it?
For the records: 0.1.19 includes PR #81, right? It's kind of confusing, because this issue is #80 :wink:

**Comment by pateketrueke** [comment]
Hi @crusy please pull v0.1.19 from NPM, it's published now.

And yeah, v0.1.19 includes both PRs.
